src/noisepy/seis/io/channelcatalog.py

```diff
@@ -304,3 +304,11 @@ def cc_parameters(cc_para, coor, tcorr, ncorr, comp):
 
 
 # TODO: A channel catalog that uses the files in the SCEDC S3 bucket: s3://scedc-pds/FDSNstationXML/
+
```

Comment:
It would be better not to have test code in the main package. My suggestion would be to leave this in `test_channelcatalog.py` and use it in all the `noisepy-seis-io` tests. For the `test_cross_correlation` in `noisepy-seis` you can create a mock, similar to how we are mocking the store [here](https://github.com/mdenolle/NoisePy/blob/136e8a6e7e24639ef8b68e8ea8277ee3af65314e/tests/test_cross_correlation.py#L47):